CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# How to contribute
+
+Thank you for looking at this. We, at Tracetest, are really excited to have you contribute on the project with us.
+
+The main lines of communication with our team is through [Github issues](https://github.com/kubeshop/tracetest/issues/new/choose) or [Discord](https://discord.gg/eBvEQRVyKX). Feel free to ask any questions!
+
+## Developing 
+
+Development can be conducted using any GoLang and Typescript compatible IDE/editor (e.g., Jetbrains GoLand, VSCode).
+
+There are four places where you develop new thing on Tracetest: on the CLI, on the API Server, on the Web UI and on the Documentation website.
+
+### CLI 
+
+Written in Golang, can be seen on the folder `./cli`. You can see what options do you have to test and build the application by typing `make help` inside the folder.
+
+To test its integration with our API, you can run a local API server on your machine by running one of our [examples](./examples/) with `docker compose`. For instance:
+```sh
+cd ./examples/tracetest-jaeger
+docker compose up -d
+```
+
+Remember that you need file pointing to that server by configuring the CLI using `tracetest configure` command or using the flag `-c` with a valid config file location, like `config.yaml`) with the following content:
+```yaml
+scheme: http
+endpoint: localhost:11633
+analyticsEnabled: false
+```
+
+### API Server
+
+Written in Golang, can be seen on the folder `./server`. You can see what options do you have to test and build the application by typing `make help` inside the folder.
+
+To run the server along the database and the Open Telemetry collector, you can run `docker compose` at the root folder of Tracetest, by executing:
+```sh
+docker compose up postgres otel-collector
+```
+
+### Web UI
+
+Written in Typescript and using Node.js tools, can be seen on the folder `./web`. You can see what options do you have to test and build the application by seeing the `package.json` scripts [here](./web/package.json).
+
+Like the CLI, to connect to the API, you can run a local API server on your machine by running one of our [examples](./examples/) with `docker compose`. For instance:
+```sh
+cd ./examples/tracetest-jaeger
+docker compose up -d
+```
+
+### Documentation website
+
+Written in Typescript and using Node.js tools using `docusaurus`, can be seen on the folder `./docs`. You can see what options do you have to test and build the application by seeing the `package.json` scripts [here](./docs/package.json).
+
+## Testing
+
+We have a great ambition to have tests verify every important corner of our code-base. It will be greatly appreciated, if your PR's include tests for the changes you propose.
```

Comment:
```suggestion
We strive to have tests verify every important corner of our code-base. It will be greatly appreciated if your PR's include tests for the changes you propose.
```